test/Hpack/ConfigSpec.hs

```diff
@@ -642,6 +645,13 @@ spec = do
         }
         )
 
+    it "allows overriding fields and takes the last value" $ do
+      withPackageConfig_ [i|
+        name: n1
+        name: n2
+        |]
+        (packageName >>> (`shouldBe` "n2"))
```

Comment:
It is still critical because otherwise merge and override (https://github.com/sol/hpack/pull/146/commits/d10a66db70aff2c225622fcdda929d15d92f7312#diff-04c6e90faac2675aa89e2176d2eec7d8R194) wouldn't work.